pkg/bff/gds.go

```diff
@@ -445,6 +448,423 @@ func (s *Server) SubmitRegistration(c *gin.Context) {
 	c.JSON(http.StatusOK, out)
 }
 
+// GetCertificates makes parallel calls to the databases to get the certificate
+// information for both testnet and mainnet. If testnetID or mainnetID are empty
+// strings, this will simply return a nil response for the corresponding network so
+// the caller can distinguish between a non registration and an error.
+func (s *Server) GetCertificates(ctx context.Context, testnetID, mainnetID string) (testnetCerts, mainnetCerts []*models.Certificate, testnetErr, mainnetErr error) {
```

Comment:
Moving the code from the former `admin.go` to here, since the methods don't have anything to do with the Admin API anymore and I couldn't think of a different name to organize them under.